pkg/controllers/management/k3supgrade/template.go

```diff
@@ -0,0 +1,163 @@
+package k3supgrade
+
+import (
+	"github.com/rancher/system-upgrade-controller/pkg/apis/upgrade.cattle.io"
+	planv1 "github.com/rancher/system-upgrade-controller/pkg/apis/upgrade.cattle.io/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubectl/pkg/describe"
+)
+
+const k3sMasterPlan = `---
+apiVersion: upgrade.cattle.io/v1
+kind: Plan
+metadata:
+  name: k3s-master-plan 
+  namespace: system-upgrade
+spec:
+  concurrency: 1
+  version: v1.17.2-k3s1
+  nodeSelector:
+    matchExpressions:
+      - {key: k3s-upgrade, operator: Exists}
+  serviceAccountName: default
```

Comment:
```suggestion
  serviceAccountName: system-upgrade # or cattle-upgrade
```

Reply:
https://github.com/rancher/rancher/pull/25667/files#diff-f70d9c6e95a58b98b200416cd10ef05eR54 
These plans are actually unused in their current form